editor.planx.uk/package.json

```diff
@@ -11,9 +11,9 @@
     "@feedback-fish/react": "^1.2.1",
     "@material-ui/core": "^4.12.4",
     "@mui/icons-material": "^5.11.16",
-    "@mui/material": "5.11.9",
-    "@mui/styles": "5.11.9",
-    "@mui/utils": "5.11.9",
+    "@mui/material": "5.13.1",
+    "@mui/styles": "5.13.1",
+    "@mui/utils": "5.13.1",
```

Comment:
This version bump fixed the annoying bug where the "Change" text within a button on the `SummaryList` wasn't correctly inheriting the font-family.

https://github.com/mui/material-ui/pull/33621